Host registration - update 'setup rex & insights' (#576)
theforeman/foreman-documentation: guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc

Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
+

. From the *Setup REX* list, select whether you want to deploy {Project} SSH keys to hosts or not.
+
If set to `Yes`, SSH keys will be installed on the registered host. The inherited value is based on the `host_registration_remote_execution` parameter.
It can be inherited e.g. from host group, operating system, organization. When overridden, the selected value will be stored on host parameter level.
+

> . From the *Setup Insights* list, select whether you want to install `insights-client` & register the hosts to Insights or not.

Reviewer: ```suggestion
. From the *Setup Insights* list, select whether you want to install `insights-client` and register the hosts to Insights.
```
The "or not" is implied in the "whether" :-)

+

> Insights tool is available for Red Hat family operating systems only, it has no effect on other OS families that do not support it.

Reviewer 2: AFAIK it's only available on RHEL, not on the Red Hat family. At least CentOS Stream 8 and CentOS Linux 7 don't have it.
```suggestion
The Insights tool is available for Red Hat Enterprise Linux only. It has no effect on other operating systems.
```

Author: I shamelessly copy/pasted the text from label help in registration form, guess we should change it there too (and the API too) ...

> Required enabled repositories on registered machine:

Reviewer: ```suggestion
You must enable the following repositories on a registered machine:
```
